### DISCUSSION TITLE
Kimai as pure Employees daily time tracking on Company & LDAP

### DISCUSSION THREAD

**Discussion by sibelle-labs** [kimai/kimai#3826] [Questions & Answers]
is it possible to use kimai as a pure Employees daily time tracking on a Company ?
Escenar: 1 company / 10 employees, then at end of Month print or Export the Timelist with worked time ( ej. 8hours x 5days x 4weeks) 

1) Wenn yes, do I still need to set up 1x company, 1x project and 1x activity?

2) ldap connection mapping.
...
            attributes:
                - { ldap_attr: employeeNumber, user_method: setUsername }
                - { ldap_attr: mail, user_method: setEmail }
                - { ldap_attr: cn, user_method: setAlias ​​}
...

Is there a method to map the "PersonalNumber" from openldap too ? 

Thanks a lot

**Comment by kevinpapst** [comment]
1: Yes. Have a look in the Store at the Kiosk plugin, simplifies life for employees, so they do not have to use the classical Web UI.

2: Yes: `{ ldap_attr: ?? PersonalNumber ??, user_method: setAccountNumber ​}`
